loadgen/contract/agent-create-vault.js

```diff
@@ -89,7 +89,7 @@ export default async function startAgent({
   // (we close over 'collateralToLock')
   async function openVault() {
     console.error('create-vault: cycle: openVault');
-    const openInvitationP = E(vaultFactory).makeLoanInvitation();
+    const openInvitationP = E(vaultFactory).makeVaultInvitation();
```

Comment:
Please use the fallback mechanism to maintain backwards compatibility with older SDKs.

For example:
https://github.com/Agoric/testnet-load-generator/blob/495ced1cc851cf30617f9403e270b73125d22807/loadgen/contract/agent-create-vault.js#L121-L124

Reply:
How old of SDK does it need to support? This "less deprecated" method has been there a while.

Reply:
So far I've been able to go back to May 2021.

Reply:
I just realized that in this case the fallback needs to be sequential instead of parallel, since both methods exist at the same time. Chaining with a `E.when` should be sufficient.

Reply:
```suggestion
    const openInvitationP = E.when(E(vaultFactory).makeVaultInvitation(), undefined, () => E(vaultFactory).makeLoanInvitation());
```

It's not as optimal, but at least it'll run.